### PR TITLE
Fix plots example code block

### PR DESCRIPTION
**Pull request by rob-luke**

### CODE REVIEW

**Comment by DilumAluthge** [comment]
I believe that the original is correct.

@KristofferC

**Comment by rob-luke** [comment]
Actually both seem to work. My appologies for adding to the noise.

**Comment by KristofferC** [comment]
Perhaps we should use the more explicit `--sysimage`?

**Comment by DilumAluthge** [comment]
> Perhaps we should use the more explicit `--sysimage`?

Sure, that might help reduce some confusion.

File: docs/src/examples/plots.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ using PackageCompiler
 create_sysimage(:Plots, sysimage_path="sys_plots.so", precompile_execution_file="precompile_plots.jl")
 ```
 
-If we now start Julia with the flag `-Jsys_plots.so` and re-time our previous commands:
+If we now start Julia with the flag `--sysimage sys_plots.so` and re-time our previous commands:
```
Comment (KristofferC) [inline docs/src/examples/plots.md]: There is a use of `-J` later in this file as well.

**Pull request merged**